test/load/locust-files/fleet_autoscaling.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from locust import HttpLocust, TaskSet, events, task
+import locust.events
+import json
+import os
+import time
+import socket
+import atexit
+
+FLEET_SIZE = 100
+DEADLINE = 30 * 60
+
+class UserBehavior(TaskSet):
+    @task
+    def scaleUpFleet(self):
+      # Create a fleet.
+      initial_size = 1
+      start_time = time.time()
+      payload = {
+          "apiVersion": "stable.agones.dev/v1alpha1",
+          "kind": "Fleet",
+          "metadata": {
+              "generateName": "fleet-simple-udp",
+              "namespace": "default"
+          },
+          "spec": {
+              "replicas": initial_size,
+              "scheduling": "Packed",
+              "strategy": {
+                  "type": "RollingUpdate"
+              },
+              "template": {
+                  "spec": {
+                      "ports": [
+                          {
+                              "name": "default",
+                              "portPolicy": "dynamic",
+                              "containerPort": 26000
+                          }
+                      ],
+                      "template": {
+                          "spec": {
+                              "containers": [
+                                  {
+                                      "name": "simple-udp",
+                                      "image": "gcr.io/agones-images/udp-server:0.5"
+                                  }
+                              ]
+                          }
+                      }
+                   }
+              }
+          }
+      }
+      headers = {'content-type': 'application/json'}
+      response = self.client.post("/apis/stable.agones.dev/v1alpha1/namespaces/default/fleets", data=json.dumps(payload), headers=headers)
+      response_json = response.json()
```

Comment:
Please run `pep8`. Install it using `pip install pep8`.
E111 indentation is not a multiple of four

Reply:
Done. Thanks.